Xamarin.PropertyEditing.Windows/PropertyButton.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using System.Windows.Data;
+using System.Windows.Input;
+using System.Windows.Shapes;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Windows
+{
+	internal class PropertyButton
+		: Control
+	{
+		static PropertyButton ()
+		{
+			FocusableProperty.OverrideMetadata (typeof (PropertyButton), new FrameworkPropertyMetadata (false));
+			DefaultStyleKeyProperty.OverrideMetadata (typeof (PropertyButton), new FrameworkPropertyMetadata (typeof (PropertyButton)));
+		}
+
+		public static readonly DependencyProperty CanSetCustomExpressionProperty = DependencyProperty.Register (
```

Comment:
Why are those three properties using `typeof (PropertyEditorControl)` and not `typeof(PropertyButton)`?